drivers/oled/oled_driver.h

```diff
@@ -200,7 +200,11 @@ void oled_write(const char *data, bool invert);
 // Advances the cursor to the next page, wiring ' ' to the remainder of the current page
 void oled_write_ln(const char *data, bool invert);
 
+// Pans the buffer to the right (or left by passing true) by moving contents of the buffer
+void oled_pan(bool left);
```

Comment:
`docs/feature_oled_driver.md` should be updated with the 2 new functions

Reply:
Thanks for the review and for pointing this out.  I've added this to the docs as indicated